node-pixels.js

```diff
@@ -102,7 +103,7 @@ function doParse(mimeType, data, cb) {
     case 'image/png':
       handlePNG(data, cb)
     break
-    
+
```

Comment:
@lukeapage Pre-emptive notice, some maintainers like keeping whitespace on empty lines. It would be a good idea to pull back these lines from `master`:

```bash
# Patch mode for checking out state from `master
# There's also `git add -p` to prevent this from occurring in the future
git checkout -p master

# Amend the current comment and update the PR
git commit --amend
git push origin master --force
```